python/ball_launcher_beepy/ball_launcher_control.py

```diff
@@ -57,8 +61,29 @@ def __init__(
         # TODO: What does this do?
         self.servo_driver1.writePin(True)
 
-    def __del__(self):
-        """Switch off motors and go to neutral orientation."""
+        try:
+            # automatic motor reset after launching
+            self.automatic_motor_reset = bool(
+                self.conf["launching_parameters"]["automatic_motor_reset"]
+            )
+            self.stirring_after_launch = bool(
+                self.conf["launching_parameters"]["stirring_after_launch"]
+            )
+        except KeyError:
+            logging.error(f"Configuration does not contain parameters.")
```

Comment:
Might be good to add the name of the missing parameter to the error message (it is included in the exception):
```suggestion
        except KeyError as e:
            logging.error(f"Configuration does not contain parameter {e}.")
```